documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -36,6 +36,14 @@ export default {
 
 Path to your custom `wrangler.toml` or `wrangler.json` config file.
 
+### handlers
```

Comment:
Can we add this option to the example config in the ## Usage section to demonstrate what it looks like when added?